autode/smiles/parser.py

```diff
@@ -254,7 +254,7 @@ def _set_implicit_hs(self):
                           'P': (3, 5), 'S': (2, 4, 6), 'F': (1,), 'Cl': (1,),
                           'Br': (1,), 'I': (1,),
                           # Aromatic atoms are distinct
-                          'b': (3,), 'c': (3,), 'n': (3,), 'o': (2,),
+                          'b': (3,), 'c': (3,), 'n': (2,), 'o': (1,),
```

Comment:
oh dear, we should have spotted that earlier. Are the other ones right? My chemistyr knowledge isn't good enough any more :(

Reply:
Indeed 😢. Perhaps boron should be 2 as well.. will check

Reply:
yeah – they were mostly all still wrong. should be correct now